drivers/wifi/nrf700x/zephyr/src/qspi/src/rpu_hw_if.c

```diff
@@ -316,7 +318,7 @@ int rpu_enable(void)
 
 int rpu_disable(void)
 {
-	gpio_pin_set_dt(&iovdd_ctrl_spec, 0); /* IOVDD CNTRL = 0 */
 	gpio_pin_set_dt(&bucken_spec, 0); /* BUCKEN = 0 */
+	gpio_pin_set_dt(&iovdd_ctrl_spec, 0); /* IOVDD CNTRL = 0 */
```

Comment:
If I remember correctly, during disable, IOVDD was to be deasserted first followed by BUCKEN- is there a reason why the order is changed?

Reply:
yes. IOVDD_CTRL has to be 0 first and then BUCKEN should be 0.

Reply:
So, is the doc wrong? Do we need to update?

Reply:
sorry the documentation in info Centre seem to be correct.